src/main/java/com/minecolonies/coremod/colony/crafting/CustomRecipe.java

```diff
@@ -234,6 +268,27 @@ public static CustomRecipe parse(@NotNull final JsonObject recipeJson)
             recipe.result = idToItemStack(recipeJson.get(RECIPE_RESULT_PROP).getAsString());
         }
 
+        if (recipeJson.has(RECIPE_ALTERNATE_PROP))
+        {
+            for (JsonElement e : recipeJson.get(RECIPE_ALTERNATE_PROP).getAsJsonArray())
+            {
+                if (e instanceof JsonElement && e.isJsonObject())
```

Comment:
`e instanceof JsonElement` is always true.

Reply:
Yeah. got it.